src/satellite_images_nso/_manipulation/nso_manipulator.py

```diff
@@ -39,17 +40,31 @@ def __make_the_crop(coordinates, raster_path, raster_path_cropped, plot):
     @param plot: Plot the results true or false
     """
 
-    geometry = [Polygon(coords) for coords in coordinates]
-    geo_area_to_use = gpd.GeoDataFrame(geometry=geometry, crs="EPSG:4326")
+    if len(coordinates) == 1:

```

Comment:
I was under the impression that this make_crop functionality already worked for multipolygons before (the downloading was the problem), so I'm not sure if we need these changes?